worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/EventAbstractionListener.java

```diff
@@ -1166,6 +1166,15 @@ public void onCauldronLevelChange(CauldronLevelChangeEvent event) {
                     item.getType() == Material.WATER_BUCKET ? Material.WATER : Material.LAVA));
             return;
         }
+
+        // handle special dispenser behavior
+        if (event instanceof BlockDispenseEvent) {
```

Comment:
Alternatively we could use the `cause` instead of the `event` here, or move the whole thing to `onBlockDispense`.

Reply:
yea i'd just move this back to the onblockdispense handler directly. this method is just for various things shared between player and dispenser "interaction"

Reply:
done